install/image-references

```diff
@@ -14,6 +14,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:openstack-machine-controllers
+  - name: openstack-machine-api-provider
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-openstack-machine-api-provider
```

Comment:
I'm still not clear how this ref gets replaced by the one from the release payload, if that's what is really happening.